share/completions/zabbix_agentd.fish

```diff
@@ -0,0 +1,43 @@
+set -l runtime userparameter_reload \
+    log_level_increase \
+    log_level_increase= \
+    log_level_decrease \
+    log_level_decrease=
+
+function __fish_has_target
+    if string match -rq '=' (commandline)
+        return 0
+    end
+    return 1
+end
+
+function __fish_string_in_command -a ch
```

Comment:
This should have a prefix to "namespace" it to these completions - see e.g. the `apt` completions with functions like `__fish_apt_no_subcommand`

Reply:
Done